force-app/main/default/classes/BotTest.cls

```diff
@@ -1,6 +1,16 @@
 @isTest
 public class BotTest {
     
+    static testMethod void testAutoBotController() {
+        //My new test method
+		Bot_Command__c bc1 = new Bot_Command__c(Sample_Utterance__c='help lightning', apex_class__c='HandlerHelpTopic', pattern__c='help (.*)');
+        insert bc1;
+        BotResponse response = BotController.submit('help lightning', null, null, null);
+        Map<String, String> session = response.session;
+        response = BotController.submit('Developer', session, null, null);
+        System.assert(response.messages[0].items.size() > 0);
+    }
+    
```

Comment:
Method `testHello` is not annotated. Remove the `testMethod` keyword and annotate with `@IsTest` instead.